File: Npc/11003848.cs
using Maple2.Model.Enum;
using Maple2.Script.Npc;

/// <summary>
/// 11003848: Condor
/// </summary>
public class _11003848 : NpcScript {
    protected override int First() {
        return 10;
    }

    // Select 0:
    // $script:0117175907009811$
    // - Let's get to business.
    protected override int Select() => 0;

    protected override int Execute(int selection) {
        switch (Id, Index++) {
            case (10, 0):
                // $script:0117175907009812$
                // - In my day, we protected the world just fine without these fancy gadgets.
                return -1;
        }
        
        return default;
    }

    protected override NpcTalkButton Button() {
        return (Id, Index) switch {
            (10, 0) => NpcTalkButton.Close,
            _ => NpcTalkButton.None,
        };
    }
}

File: Npc/11003848.py
""" 11003848: Condor """
from npc_api import Option, Script
import random


class Main(Script):
    def first(self) -> int:
        return 10

    def select(self) -> int:
        return 0

    def __0(self, pick: int) -> int:
        # $script:0117175907009811$
        # - Let's get to business.
        return None # TODO

    def __10(self, pick: int) -> int:
        # $script:0117175907009812$
        # - In my day, we protected the world just fine without these fancy gadgets.
        return -1

    def button(self) -> Option:
        if (self.state, self.index) == (10, 0):
            return Option.CLOSE
        return Option.NONE
